enterprise/core-edge/src/main/java/org/neo4j/coreedge/server/CoreEdgeClusterSettings.java

```diff
@@ -94,12 +94,14 @@ public String toString()
             setting( "core_edge.log_shipping_max_lag", INTEGER, "256" );
 
     @Description("Size of the RAFT in queue")
+    @Internal
     public static final Setting<Integer> raft_in_queue_size =
             setting( "core_edge.raft_in_queue_size", INTEGER, "64" );
 
     @Description("Largest batch processed by RAFT")
+    @Internal
     public static final Setting<Integer> raft_in_queue_max_batch =
-            setting( "core_edge.raft_in_queue_max_batch", INTEGER, "16" );
+            setting( "core_edge.raft_in_queue_max_batch", INTEGER, "64" );
```

Comment:
It doesn't make sense to put max_batch > queue_size, but I don't know how to link them really. I have marked both settings as @Internal to make it clear that they are not generally to be configured by users.